networkx/generators/social.py

```diff
@@ -21,6 +21,13 @@ def karate_club_graph():
     belongs, either 'Mr. Hi' or 'Officer'. Each edge has a weight based on the
     number of contexts in which that edge's incident node members interacted.
 
+    The dataset is derived from the 'Club After Split From Data' column of Table 3 in the source paper.
+    This was in turn derived from the 'Club After Fission' column of Table 1 in the same paper.
+    Note that the nodes are 0-indexed in NetworkX, but 1-indexed in the paper
+    (the 'Individual Number in Matrix C' column of Table 3 starts at 1).
+    This means, for example, that ``G.nodes[9]["club"]`` returns 'Officer',
+    which corresponds to row 10 of Table 3 in the paper.
```

Comment:
```suggestion
    The dataset is derived from the 'Club After Split From Data' column of Table 3 
    in the source paper.
    This was in turn derived from the 'Club After Fission' column of Table 1 in the
    same paper. Note that the nodes are 0-indexed in NetworkX, but 1-indexed in the
    paper (the 'Individual Number in Matrix C' column of Table 3 starts at 1). This
    means, for example, that ``G.nodes[9]["club"]`` returns 'Officer', which
    corresponds to row 10 of Table 3 in the paper.
```

Reply:
@mbbolger I didn't make any changes to your documentation, it's just a suggestion to comply with 80 col as @dschult  says.

I hope it helps :thumbsup:

Reply:
Sorry for the delay, just catching up on this. Thanks for the help @miguelcsx , I'll commit that change!